### ISSUE TITLE
[Label]: missing accessibility properties for "required" attribute

### ISSUE THREAD

**Issue by anshul-031** [SAP/ui5-webcomponents-react#4456]
### Describe the bug

Screen readers don't read "required" for the "Label" web component that are enabled required

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-label--docs

### Reproduction steps

1. Set property 'required' of Label as true.
2. Asterik(*) will come for the Label
3. The screen render don't read out the 'required' word

### Expected Behaviour

It should read out 'required' if some field is set as required

### UI5 Web Components for React Version

1.8.0

### UI5 Web Components Version

1.10.3

### Browser

Chrome, Edge

### Operating System

Windows

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
